pkg/api/aim/query/query.go

```diff
@@ -38,12 +38,12 @@ type parsedQuery struct {
 	conditions []clause.Expression
 }
 
-type attributeGetter func(attr string) (any, error)
-
 type callable func(args []ast.Expr) (any, error)
 
 type subscriptSlicer func(index ast.Slicer) (any, error)
 
+type attributeGetter func(attr string) (any, error)
+
```

Comment:
any reason why you moved `attributeGetter` here?

Reply:
hmmm, should it not be before `subscriptSlicer` then?

Reply:
no no, not alphabetical - it is pain, just from short to long.

Reply:
but yep, lets sort it fully.

Reply:
but they have the same length for their names, and then `subscriptSlicer` has a longer signature — I'm cool with going with non-alphabetical and go for length, but we need to be consistent!

Reply:
yep, the use this way. agreed. done.